test/hello_world/Dockerfile

```diff
@@ -1,9 +1,34 @@
-FROM ros:indigo-ros-base
+FROM ros:indigo
```

Comment:
Both of these Dockerfiles can be collapsed into one by using a multi-stage build: https://docs.docker.com/develop/develop-images/multistage-build/

```
FROM ros:indigo as base
...

FROM base as buggy
...
```

Reply:
You mean having one Dockerfile instead of two?